submissions/sayopaul/Week1/fizz.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+	for($i=1; $i<=100; $i++){
+
+		if($i % 15 === 0 ){
+			$output = "FizzBuzz".PHP_EOL;	
+
+		}elseif($i % 3 === 0 ){
+			$output = "Fizz".PHP_EOL;
+
+		}elseif($i % 5 === 0 ){
+			$output = "Buzz".PHP_EOL;
+
+		}else{
+		    $output = $i.PHP_EOL;
+		}
+		echo $output;
+
+	}
+
+
```

Comment:
too much unnecessary new lines at the end of file

Reply:
I have removed them sir

Reply:
Thanks for the update. I prefer you refer to me as @codedentwickler, please don't call me sir.